real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/localizers/RobobuggyKFLocalizer.java

```diff
@@ -0,0 +1,356 @@
+package com.roboclub.robobuggy.nodes.localizers;
```

Comment:
This file suffers from inconsistent naming with our static analysis tools. Either we need to disable naming checks or fix the namings in here